yarn.lock

```diff
@@ -4791,6 +4791,18 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@graphql-yoga/redis-event-target@npm:2.0.0":
+  version: 2.0.0
+  resolution: "@graphql-yoga/redis-event-target@npm:2.0.0"
+  dependencies:
+    "@graphql-yoga/typed-event-target": ^2.0.0
+    "@whatwg-node/events": ^0.1.0
+  peerDependencies:
+    ioredis: ^5.0.6
```

Comment:
Oh I just noticed that `redis-event-target` relies on IORedis.

If I just support IOredis, I could get rid on the generic interface and the api package add to the realtime setup handler.

Reply:
Thoughts on supporting IORedis and Redis?

Reply:
I feel like if the graphql package is preferring to use `ioredis` then I think we should just do the same. Although I don't claim to have any particular insight, it just feels natural to follow the lead of the tool we're building on top of. We can explore being fully generic in the future if there is demand from users? 

This fires up my internal questions about node and dependencies. I wish there was some sort of way to say "I'm using this feature" and then that to cause conditional dependencies under the hood. I think that sort of ability doesn't exist and it is just left to the user to specify that sort of dependency on their own. Is this what rust's "features" are :man_shrugging:? Anyway this is all getting off topic.

Reply:
Now that I look how the redis event target works. I dthyresson@yahoo.com k I am going to simplify and just use IORedis that comes from yoga and its subscription support.

https://github.com/dotansimha/graphql-yoga/blob/252b020f4d485553547f816da4cfb5958424f1fe/packages/event-target/redis-event-target/src/index.ts#L3

First: no real benefit from using the node Redis client over IORedis

2nd: this way the types can be simplified and always will be in sync with Yoga


3rd: Uostash redis doesn’t support subscribe or connect so can’t use anyway.

https://docs.upstash.com/redis/features/restapi#rest---redis-api-compatibility

4th Can use Redis clusters this way

https://redis.io/docs/reference/cluster-spec/

Reply:
I refactored to better take use of CreateRedisEventTargetArgs so that the Redis types are compatible and can use cluster.


ioerdis in framework only needs to be a dev dependency for types